### PR TITLE
chore(quarkus-integration): Extend Quarkus Compatibility

### PR DESCRIPTION
**Pull request by psavidis**
- camunda 7.21.x with Quarkus 3.8.x

Related-to: https://github.com/camunda/camunda-bpm-platform/pull/3785

### CODE REVIEW

**Comment by psavidis** [comment]
## Notes for the Reviewer

- The Matrix has been updated to mention **Quarkus 3.8** instead of **3.7**
- The latest code in the PR is using **Quarkus 3.7**, 3.8 will be released at the end of February.
- This will save us from updating it twice. Still if you think its a risk, feel free to include it in your feedback

**Review by yanavasileva — COMMENTED** [review]
Having the correct minor version for the alpha will avoid misunderstanding among users.

File: content/user-guide/quarkus-integration/version-compatibility.md [hunk 1]
```diff
@@ -35,6 +35,10 @@ Only these default combinations are recommended (and supported) by Camunda.
     <td>7.20.x</td>
     <td>3.2.x.Final</td>
   </tr>
+  <tr>
+    <td>7.21.x</td>
+    <td>3.8.x.Final</td>
```
[the remark this answers is not in the document]
Comment (psavidis) [reply content/user-guide/quarkus-integration/version-compatibility.md]: ❓ @yanavasileva Now the version needs to be upgraded to `7.21.0-alpha4` and one time additionally for each alpha or minor released. Should we just add `7.21.0`?

[the remark this answers is not in the document]
Comment (yanavasileva) [reply content/user-guide/quarkus-integration/version-compatibility.md]: Ok, fine with me.

**Review by yanavasileva — APPROVED** [review]

**Pull request merged**